src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -114,6 +114,19 @@ void GeotaggedImagesPlugin::Load(sensors::SensorPtr sensor, sdf::ElementPtr sdf)
         _maxZoom = sdf->GetElement("maximum_zoom")->Get<float>();
     }
 
+    if (sdf->HasElement("stream_uri")) {
+        _streamURI = sdf->GetElement("stream_uri")->Get<int>();
+    }
+    if (sdf->HasElement("system_id")) {
+        _systemID = sdf->GetElement("system_id")->Get<int>();
+    }
+    if (sdf->HasElement("component_id")) {
+        _componentID = sdf->GetElement("component_id")->Get<int>();
+    }
+    if (sdf->HasElement("component_id")) {
+        _camPort = sdf->GetElement("cam_port")->Get<int>();
```

Comment:
`cam_port` seems a misleading name since it is the udp port of the mavlink stream. Probably be more explicit on naming such as `mavlink_udp_port`

Reply:
@mzahana I would prefer removing the `cam` since this port is not a udp port of the camera, but the camera manager

Reply:
Noted.

Reply:
Okay, lets leave it for now then